src/sass/components/_tour.scss

```diff
@@ -37,24 +37,73 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  top: -($spacing-small);
+
+  .epic-progress & {
+    right: -($spacing-small);
+    top: -($spacing-medium);
+  }
 
   .logo-popover & {
     top: unset;
   }
 
+  .metecho-breadcrumb & {
+    top: -($var-spacing-x-small);
+  }
+
+  .next-steps & {
+    left: 15.5em;
+  }
+
+  .project-placeholder & {
+    top: $spacing-medium;
+  }
+
+  .right-edge & {
+    right: 0;
+    top: -($var-spacing-x-small);
+  }
+
+  .vertical-right & {
+    right: -($spacing-x-small);
+    top: -($spacing-medium);
+  }
+
+  .submit-btn & {
+    right: -($spacing-xxx-small);
+    top: -($spacing-large);
+  }
+
+  .task-create-btn & {
+    top: -($var-spacing-large);
+  }
+
   .page-title & {
-    top: 25px;
+    left: $spacing-large;
+    top: $var-spacing-medium;
+
+    @media (min-width: $mq-medium) {
+      left: unset;
+    }
   }
 }
 
+.metecho-breadcrumb ol {
```

Comment:
Do you know @jgerigmeyer if we ever did truncate each breadcrumb? I looked at staging and we don't do it there. 

@dvdherron One option (new story I assume) I see SLDS has an overflow option https://www.lightningdesignsystem.com/components/breadcrumbs/#With-Overflow 

Another idea **If we want to truncate each `<li>` separately**, which I think is better usability, we would need to do something like 
- updating this line to `.metecho-breadcrumb li` instead of `ol`
- overriding the `<a>` to not have `display: block` 
- optional:  `.metecho-breadcrumb li:not(:first-of-type)` so it doesn't truncate home.
- we'd need to add a way to target the last item, which isn't a link, to tell that div in that last li not to display block... `.metecho-breadcrumb .slds-breadcrumb__item:last-of-type > div { display: inline (or contents) }` ... that is unless we can change it to a span?

Reply:
> Do you know @jgerigmeyer if we ever did truncate each breadcrumb? I looked at staging and we don't do it there.

I'm not sure -- I'm guessing we didn't?

Reply:
I tried truncating the `<li>`'s separately yesterday but having `overflow-hidden` cut of parts of the list markers and made them all really small @stacyk. I'll play around with this some more in a bit after I reposition the popovers